designdocs/delegation.md

```diff
@@ -69,12 +69,12 @@ Adds the `MsaId` in the payload as a delegate, to an MSA owning `delegator_msa_i
   * Parameters:
       1. `payload`: authorization data signed by the delegating account
           1. `data` - this is what the MSA owner must sign and provide to the delegate beforehand.
-              * `delegate_msa_id` - the delegate's `MsaId`, i.e. the caller's `MsaId`
```

Comment:
Generic info:
Delegate is Provider now and accordingly, naming conventions have been updated